src/Routing/Asset.php

```diff
@@ -254,6 +254,10 @@ public static function assetTimestamp(string $path, $timestamp = null): string
             }
             $segments = explode('/', ltrim($filepath, '/'));
             $plugin = Inflector::camelize($segments[0]);
+            if (!Plugin::isLoaded($plugin) && count($segments) > 1) {
```

Comment:
Can we comment was is happening in this section below the timestampEnabled check.